### PR TITLE
redwood: initial support

### PR DESCRIPTION
**Pull request by thepriyanshujangid**

### CODE REVIEW

File: redwood.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+			"oem": "Poco",
+			"device": "Poco X5 Pro 5G",
+			"filename": "crDroidAndroid-15.0-20241107-redwood-v11.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/redwood/11.x/crDroidAndroid-15.0-20241107-redwood-v11.0-BETA.zip/download",
+			"timestamp": 1730996903,
+			"md5": "5c890b8e2f78d75ad6bb4e5883d2fdca",
+			"sha256": "5239c75bf990c24cf13f198704892a0d0be36cd69fd5867124c46eddc06ed4a4",
+			"size": 1606222282,
+			"version": "11.0",
+			"buildtype": "Monthly",
```
Comment (gwolf2u) [inline redwood.json]: BETA

File: redwood.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			"forum": "https://xdaforums.com/t/rom-15-0-redwood-crdroid-v11-0-official.4672931/",
```
Comment (gwolf2u) [inline redwood.json:15]: Remove telegram links on forum and use new support page url
Mandatory to comply with XDA rules

File: redwood.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+			"gapps": "https://nikgapps.com/",
+			"firmware": "https://xiaomifirmwareupdater.com/archive/firmware/redwood/",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "https://sourceforge.net/projects/crdroid/files/redwood/11.x/recovery/",
+			"paypal": "https://paypal.me/thepriyanshujangid",
+			"telegram": "https://t.me/redwoodroms",
+			"dt": "https://github.com/crdroidandroid/android_device_xiaomi_redwood",
+			"common-dt": "https://github.com/crdroidandroid/android_device_xiaomi_sm8350-common",
+			"kernel": "https://github.com/crdroidandroid/android_kernel_xiaomi_sm8350"
```
Comment (gwolf2u) [inline redwood.json:25]: missing 15.0

File: redwood_changelog.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Highlights & Device Specific Changes:
+Build type: Monthly
```
Comment (gwolf2u) [inline redwood_changelog.txt]: BETA

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: redwood.json (hunk 3 above)
Comment (thepriyanshujangid) [reply redwood.json:25]: missing 15.0 of what sir ?

Comment (gwolf2u) [reply redwood.json:25]: tagged this line
this line is a repo link
so to the repo

Comment (thepriyanshujangid) [reply redwood.json:25]: 15.0  kernel branch right ?
if yes may i  rename 14.0-redwood > 15.0-redwood  ?

Comment (gwolf2u) [reply redwood.json:25]: - yes
- no, create a new branch

**Pull request closed without merging**